src/app/clusters/on-off-server/on-off-server.h

```diff
@@ -87,6 +87,12 @@ struct OnOffEffect
     OnOffEffect(
         chip::EndpointId endpoint, OffWithEffectTriggerCommand offWithEffectTrigger,
         uint8_t effectIdentifier = static_cast<uint8_t>(EMBER_ZCL_ON_OFF_EFFECT_IDENTIFIER_DELAYED_ALL_OFF),
```

Comment:
What about type-safety for this one, where it _is_ possible?

Reply:
fixed in #12813